.github/workflows/patchworks.yaml

```diff
@@ -8,6 +8,13 @@ on:
   pull_request:
     branches:
       - patchworks-ci
+  workflow_dispatch:
+    branches:
+      - patchworks-ci
+    inputs:
+      backup_time:
+        description: 'Timestamp (UTC) 15 min before start time'
```

Comment:
What does the start time refer to here? If I have a patch where I know its timestamp, can I just plug in that timestamp and get the 15 minute range around it?

Something similar to this where it'll auto-round the timestamp for me? (not tested, so the time ranges are probably wrong)
```
TZ=UTC date -d "@$((($(${{ inputs.backup_time }} +%s) + 450) / 900 * 900))" '+%Y-%m-%dT%H:%M:%S' > date_recent.txt
TZ=UTC date -d "@$((($(${{ inputs.backup_time }} +%s) - 450) / 900 * 900))" '+%Y-%m-%dT%H:%M:%S' > date_old.txt
TZ=UTC date -d "@$((($(${{ inputs.backup_time }} +%s) - 1350) / 900 * 900))" '+%Y-%m-%dT%H:%M:%S' > date_older.txt
```

Reply:
Updated so that if we know the patch timestamp, can plug in that timestamp and get the 15 min range around it
Tested with
`date -d "@$((($(date -d 2023-09-28T16:32:28 +%s) + 450) / 900 * 900))" '+%Y-%m-%dT%H:%M:%S'`